perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -721,7 +721,8 @@ public void setAttributes(PerunSession sess, Facility facility, List<Attribute>
 				}
 			}
 		}
-		//if checkAttributesSemantics fails it causes rollback so no attribute will be stored
+		//if checkAttributesSyntax or checkAttributesSemantics fails it causes rollback so no attribute will be stored
```

Comment:
Please keep comments separated. CheckAttributesSemantics will be moved to a different part of the set method later, so it is better to keep the comments separated.

Reply:
Okay, the comments are now separated.